main/integration-tests/__test__/api-tests/users/bulk-add-users.test.js

```diff
@@ -0,0 +1,67 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const { runSetup } = require('../../../support/setup');
+const errorCode = require('../../../support/utils/error-code');
+
+describe('Create user scenarios', () => {
+  let setup;
+  let defaultUser;
+  let username;
+
+  beforeAll(async () => {
+    setup = await runSetup();
+    const adminSession = await setup.defaultAdminSession();
+    username = await setup.gen.username();
```

Comment:
setup.gen.usrename() is not an async function.

you can just do username = setup.gen.username()